src/bundle/Resources/public/scss/ui/modules/sub-items-list/_grid.view.item.scss

```diff
@@ -1,20 +1,23 @@
 .c-grid-view-item {
-    position: relative;
-    overflow: hidden;
-    margin-bottom: calculateRem(20px);
-    cursor: pointer;
-    color: $ibexa-color-font;
-    text-decoration: none;
-    transition: all 0.3s $ibexa-admin-transition;
-    background: $ibexa-white;
-    border: calculateRem(1px) solid rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+    padding: calculateRem(16px);
+    height: calculateRem(238px);
```

Comment:
Are you sure we want to have fixed height? What if title or sth will be multiline?